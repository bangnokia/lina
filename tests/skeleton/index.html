<html>
<head>
    <title>Hello world</title>
</head>
<body>
    <h1>Hello world</h1>
    <div>
        &lt;p&gt;Hello John Doe&lt;/p&gt;
    </div>
</body>
</html>
