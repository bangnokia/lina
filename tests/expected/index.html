<html>
<head>
    <title>Home page</title>
</head>
<body>
    <h1>Home page</h1>

    <ul>
                    <li><a href="/foo">[2020-11-01] Foo</a></li>
                    <li><a href="/hello-world">[2020-10-30] Hello world</a></li>
            </ul>
</body>
</html>
